lang/fr/main.json

```diff
@@ -57,15 +62,15 @@
     "Invalid Captcha": "Captcha invalide",
-    "Is a digital signature the same as a digitized signature?": "Une signature numérique est-elle identique à une signature numérisée ?",
+    "Is a digital signature the same as a digitized signature?": "Is a digital signature the same as a digitized signature?",
```

Comment:
Some translations reverted from the target language back to English, is this normal?